src/components/Layout/Header/NavBar/UserMenu.tsx

```diff
@@ -50,7 +52,11 @@ export type WalletConnectedProps = {
   onDisconnect: () => void
   onSwitchProvider: () => void
   onClose?: () => void
-} & Pick<InitialState, 'walletInfo' | 'isConnected' | 'connectedType'>
+  walletInfo: {
+    icon: ComponentWithAs<'svg', IconProps> | string
+    name: string
```

Comment:
The only two properties we were concerned about here, allows compatibility between types